aci/resource_rest.go

```diff
@@ -18,23 +19,26 @@ const ErrDistinguishedNameNotFound = "The Dn is not present in the content"
 
 func resourceAciRest() *schema.Resource {
 	return &schema.Resource{
-		Create: resourceAciRestCreate,
-		Update: resourceAciRestUpdate,
-		Read:   resourceAciRestRead,
-		Delete: resourceAciRestDelete,
+		Create:        resourceAciRestCreate,
+		Update:        resourceAciRestUpdate,
+		Read:          resourceAciRestRead,
+		Delete:        resourceAciRestDelete,
+		CustomizeDiff: resourceAciRestCustomizeDiff,
```

Comment:
I don't think we need this CustomizeDiff function here as we don't need to check for the changes in the Dn. Please remove this function as well.

Reply:
I will remove this function when I address the thread on the force new on the dn attribute